src/main/java/org/configureme/sources/FileLoader.java

```diff
@@ -77,7 +77,7 @@ public String getContent(final ConfigurationSourceKey key){
 			log.info("load configuration from file: " + f.getAbsolutePath());
 		try{
 			if (!f.exists())
-				return getContentFromJar(f.getName());
```

Comment:
When using a config file `distributeme/rights-routing`, `File.getName()` will only return `rights-routing`, so the path is lost